includes/sanitizers/class-amp-tag-and-attribute-sanitizer.php

```diff
@@ -2627,10 +2627,10 @@ private function remove_node( DOMElement $node ) {
 		/**
 		 * Parent.
 		 *
-		 * @var DOMNode $parent
+		 * @var DOMElement|null $parent
 		 */
 		$parent = $node->parentNode;
-		if ( $node && $parent ) {
```

Comment:
Yes, Because as per function and param type declaration type of `$node` is `DOMElement`. So, It assumes that $node will never be `null`.


Error in PHPstan
```
 ------ --------------------------------------------------------------- 
  Line   includes/sanitizers/class-amp-tag-and-attribute-sanitizer.php  
 ------ --------------------------------------------------------------- 
  2633   Left side of && is always true.                                
 ------ ---------------------------------------------------------------
```